1.6/Dockerfile.rhel7

```diff
@@ -46,9 +45,11 @@ RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
     mv oc /usr/local/bin && \
     rm oadm && \
     rm openshift && \
-    chmod -R og+rw /opt/openshift
+    mkdir -p /opt/openshift && chmod -R og+rw /opt/openshift && \
+    mkdir -p /usr/share/jenkins && chmod -R og+rw /usr/share/jenkins
 
-VOLUME ["/var/lib/jenkins"]
+COPY ./contrib/openshift /opt/openshift
+COPY ./contrib/jenkins /usr/local/bin
```

Comment:
what's the permissions of config.xml?

random uids will no longer be able to create new files under the configuration directory, not sure if that could be problematic, i'd have to revisit the directory structure.

what about the "jobs" directory?  isn't that going to also not be writable by uid 1001 (or any random uid) which means the jenkins process won't be able to write changes or jobs or create new jobs?